cmd/interactsh-client/main.go

```diff
@@ -44,6 +47,7 @@ func main() {
 
 	flagSet.CreateGroup("config", "config",
 		flagSet.StringVar(&cliOptions.Config, "config", defaultConfigLocation, "flag configuration file"),
+		flagSet.DynamicVar(&cliOptions.PdcpAuth, "auth", "true", "configure projectdiscovery cloud (pdcp) api key"),
```

Comment:
Why not use `BoolVarP` instead?

Reply:
With the same option they can also pass api key: `./interactsh-client -auth=xxx`, for this purpose we've used dynamicvar here.

Reply:
If this what we're aiming for, we should be more explicit; a more descriptive helper text + example in readme file